modules/i18n/src/models/namespaces/console-ns.ts

```diff
@@ -3733,6 +3733,19 @@ export interface ConsoleNS {
                     loginAttemptsSecurity : {
                         name: string;
                         connectors: {
+                            adminSessionAdvisoryBannerHandler: {
```

Comment:
What is the usage of these strings? couldn't see any usage in this PR though. Maybe you are planning to send the configuration UI impl in a follow-up PR?

Reply:
According to the governance connector implementation, you only need to define the newly created connector under the relavant connectorCategory. Then the connectors defined under connectorCategories are being automatically displayed in the relavant pages. 

Please refer [const resolveConnectorTitle = (connector: GovernanceConnectorInterface): ReactNode => {](https://github.com/wso2/identity-apps/blob/74b275743a60eb7032843f23bf55f157ec35aebe/apps/console/src/features/server-configurations/components/governance-connectors/dynamic-governance-connector.tsx#L207) for more details.